src/vs/editor/common/config/commonEditorConfig.ts

```diff
@@ -207,6 +207,32 @@ const editorConfiguration: IConfigurationNode = {
 			'default': 'on',
 			'description': nls.localize('lineNumbers', "Controls the display of line numbers. Possible values are 'on', 'off', and 'relative'. 'relative' shows the line count from the current cursor position.")
 		},
+		'editor.lineNumberInterval': {
```

Comment:
IMHO, the introduction of the OR type makes this setting too difficult to configure. It is technically correct, but does not align with the other editor settings.

> I would remove the usage of the OR type.